src/htmlContent/video-view.html

```diff
@@ -0,0 +1,22 @@
+<div class="video-view">
+    <div class="video-data"></div>
+    <div class="video-wrapper">
+        <video controls>
+          <source src="{{ videoUrl }}" type="video/mp4">
+        </video>
+    </div>
+    <p>Embed Code</p>
+<pre>
```

Comment:
Yeah but that's the code someone is supposed to copy and paste into their HTML page.

```
<video controls>
  <source src="/7/projects/30/preview-modes.mp4" type="video/mp4">
  Your browser doesn't support HTML5 video tag.
</video>
```
I'm okay removing it, but just want to make sure we're talking about the same thing.